api/migrations/versions/54aba10497f4_.py

```diff
@@ -0,0 +1,36 @@
+"""empty message
+
+Revision ID: 54aba10497f4
+Revises: d7f3adff2c9f
+Create Date: 2020-02-14 20:24:03.644194
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '54aba10497f4'
+down_revision = 'd7f3adff2c9f'
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('file',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('file_name', sa.String(length=50), nullable=False),
+    sa.Column('mime_type', sa.String(length=20), nullable=False),
+    sa.Column('guid', sa.String(length=50), nullable=False),
+    sa.PrimaryKeyConstraint('id'),
+    sa.UniqueConstraint('file_name'),
+    sa.UniqueConstraint('guid')
+    )
+    op.drop_column(u'application_form', 'deadline')
```

Comment:
So when creating a new migration something to be mindful of is to watch for fields appearing that have nothing to do with the changes you've made. For example this one on `application_form`. This field was recently dropped in another PR. I think what happened was that you merged in the latest changes but you never upgraded your local version of your database resulting in it appearing to alembic that the database and the application_form model were out of sync. Normally alembic shouldn't let you make a migration if your local database isn't up to date but seemed like that didn't happen for some reason. So I think for now just back space this line.